backend/src/ml_space_lambda/group/lambda_functions.py

```diff
@@ -140,7 +141,7 @@ def add_users(event, context):
                 GroupUserModel(
                     group_name=group_name,
                     username=username,
-                    permissions=[Permission.COLLABORATOR],
+                    permissions=[],
```

Comment:
I think we can just leave this argument out, `permissions` already defaults to `[]`